lib/flight_scheduler/job_runner.rb

```diff
@@ -68,9 +114,8 @@ def run_job(job_id, env, *arguments, **options)
     # * Returns an Async::Task that can be `wait`ed on.  When the returned
     #   task has completed, the subprocess will have been sent a `TERM`
     #   signal.
-    def cancel_job(job_id)
-      process = FlightScheduler.app.job_registry[job_id]
-      if process && process.running?
+    def cancel
+      if child && child.running?
         Async do
           process.kill
```

Comment:
Where does `process` come from?  Should it be `child`?

Reply:
opps, thought I got them all:
https://github.com/openflighthpc/flight-scheduler-daemon/pull/4/commits/07d6273e71504357f9a4c049532d94783f071744